helpers/bins.py

```diff
@@ -1,9 +1,11 @@
 from array import array
+import itertools
 
 jetEtaBins = [0., 1.3, 2.5, 3., 3.5, 4., 5.]
 egEtaBins = [0., 1.479, 2.5]
 muEtaBins = [0., 0.83, 1.24, 2.4]
 muEMTFBins = [1.24, 1.6, 2.1, 2.4]
+tauEtaBins = [0., 1.5, 2.1]
```

Comment:
The usual  |eta| boundary for barrel-endcap is rather 1.3. Can you adapte? Thanks !

Reply:
This bins were used during the testing. Not being used now. I have commented it out. It might be needed later for offline L1 Tau studies